### PR TITLE
[tsa] MCH-PE20.08-UP01 for RH7.9

### PR DESCRIPTION
**Pull request by kraushm**
The changes have been deployed as new production software stack during the roll-out of the OS upgrade to RH7.9.

### CODE REVIEW

**Comment by kraushm** [comment]
retest this please

File: jenkins/Machines.groovy [hunk 1]
```diff
@@ -33,9 +33,9 @@ def pilatus = [name: 'pilatus',
 def tsa = [name: 'tsa',
            archs: [],
            buildPath: '/tmp/$USER/easybuild/build',
-           unusePath: '/apps/arolla/UES/jenkins/RH7.7/MCH-PE20.06/generic/easybuild/modules/all',
+           unusePath: '',
            modulesProduction: '',
            modulesUnuseProduction: '',
-           prefixProduction: '/apps/arolla/UES/jenkins/RH7.7/MCH-PE20.08/generic/easybuild']
+           prefixProduction: '/apps/arolla/UES/jenkins/RH7.9/MCH-PE20.08-UP01/generic/easybuild/modules/all']
```
Comment (lucamar) [inline jenkins/Machines.groovy:39]:
```suggestion
           prefixProduction: '/apps/arolla/UES/jenkins/RH7.9/MCH-PE20.08-UP01/generic/easybuild']
```

Comment (lucamar) [reply jenkins/Machines.groovy:39]: @kraushm I think `/modules/all` should not appear in the `prefix`, since it is added by EasyBuild

**Review by lucamar — CHANGES_REQUESTED** [review]
Thanks @kraushm, please see my suggested edit

File: jenkins/Machines.groovy (hunk 1 above)
Comment (kraushm) [reply jenkins/Machines.groovy:39]: Indeed, my bad.

**Review by lucamar — APPROVED** [review]
Thanks @kraushm

**Pull request merged**